chrome/content/misc.js

```diff
@@ -19,7 +19,16 @@ function NostalgyDebug(aText)
 
 function NostalgyStopEvent(ev) {
   ev.preventDefault();
-  // ev.stopPropagation();
+  //ev.stopPropagation();
+  ev.stopImmediatePropagation();
+//   alert("stop ev");
+}
+
+function NostalgyStopEventImmediately(ev) {
+  ev.preventDefault();
+  //ev.stopPropagation();
+  ev.stopImmediatePropagation();
+//   alert("stop ev");
```

Comment:
`NostalgyStopEvent` and `NostalgyStopEventImmediately` have the same content. Is this intended?

Also, I would suggest removing the commented out `alerts`.